docs/docs/reference/new-types/intersection-types.md

```diff
@@ -11,18 +11,18 @@ The type `S & T` represents values that are of the type `S` and `T` at the same
 
 ```scala
 trait Resettable {
-  def reset(): this.type
```

Comment:
It needs a result type for sure. If not `this.type` then `Unit`.

Reply:
sorry, was thinking in the wrong language. I thought absent type in trait = Unit. I'am going to use Unit I thought this.type could confuse beginners